WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -488,6 +530,7 @@ public boolean matchesSafely(View view) {
         };
     }
 
+
```

Comment:
Probably don't need the extra whitespace here?

Reply:
Done